From 5412077c398dec74321388fe6e593a44c4c80de6 Mon Sep 17 00:00:00 2001
From: echo <fei.geng@windriver.com>
Date: Tue, 28 Apr 2009 03:11:06 +0000
Subject: [PATCH] Fix perl install directory to /usr/bin

Upstream-Status: Inappropriate [configuration]

Add back this patch. Without this patch, apxs's shebang will use
perl under hosttools, which can be too long for shebang, and cause
error:
bad interpreter: No such file or directory

Signed-off-by: Changqing Li <changqing.li@windriver.com>

---
 configure.in | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/configure.in b/configure.in
index d828512..be7bd25 100644
--- a/configure.in
+++ b/configure.in
@@ -855,10 +855,7 @@ AC_DEFINE_UNQUOTED(SERVER_CONFIG_FILE, "${rel_sysconfdir}/${progname}.conf",
 AC_DEFINE_UNQUOTED(AP_TYPES_CONFIG_FILE, "${rel_sysconfdir}/mime.types",
 	[Location of the MIME types config file, relative to the Apache root directory])
 
-perlbin=`$ac_aux_dir/PrintPath perl`
-if test "x$perlbin" = "x"; then
-    perlbin="/replace/with/path/to/perl/interpreter"
-fi
+perlbin='/usr/bin/perl'
 AC_SUBST(perlbin)
 
 dnl If we are running on BSD/OS, we need to use the BSD .include syntax.
